packages/redux-query-react/flow-test/hooks/use-request-on-demand.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import * as React from 'react';
+
+import useRequestOnDemand from '../../src/hooks/use-request-on-demand';
+
+const Card = () => {
+  const [{ isPending }, request] = useRequestOnDemand((force: boolean) => ({
+    url: '/api',
+    force,
+  }));
+
+  return (
+    <div>
+      {isPending === true ? 'loading…' : 'loaded'}
+      {/* $FlowFixMe expected */}
+      <button onClick={() => request('false')}>Trigger</button>
```

Comment:
false as a string?

Reply:
this is a test to prove that the flow typing is working. it's expecting line 17 to error because its a string and not a bool. Could be any string or any non bool value.